workspace/apps/pidp/src/app/features/auth/pages/login/login.page.html

```diff
@@ -7,64 +7,101 @@ <h1 class="title">
             </p>
-          </section>
-          <section class="title-bcgov">
+          </section> -->
+      <!-- <section class="title-bcgov">
             <div class="title-bcgov-logo">
               <picture>
                 <source
                   type="image/webp"
                   srcset="/assets/images/BC_HLTH_H_RGB_rev.webp" />
                 <source
                   type="image/png"
-                  srcset="/assets/images/BC_HLTH_H_RGB_rev.png" />
+                  srcset="/assets/images/login-image-desktop.png" />
                 <img
                   title="BC Government Logo"
                   loading="lazy"
                   aria-label="BC Ministry of Health Logo" />
               </picture>
             </div>
-          </section>
-        </div>
-      </header>
+          </section> -->
+      <!-- </div> -->
+      <div class="background-image-container">
+        <img
+          ngSrc="/assets/images/login-image-giant-1920x1280.jpg"
+          priority
+          fill />
+      </div>
+
       <section class="welcome-section">
         <div class="welcome-container">
-          <div *ngIf="isPidpLogoVisible" class="welcome-pidp-logo">
-            <img
-              src="/assets/images/pidp-logo-blue.svg"
-              title="PIDP Logo"
-              loading="lazy"
-              aria-label="PIDP Logo" />
+          <div class="title-content-container">
+            <h1 class="title">
+              <span class="title-text"
+                ><strong>OneHealthID Service</strong></span
+              >
+            </h1>
+            <section class="title-site-description">
+              <p>
+                A suite of identity-related products and services for healthcare
+                providers in BC.
+              </p>
+            </section>
           </div>
           <ng-container *ngTemplateOutlet="login"></ng-container>
         </div>
-      </section>
-      <section class="systems-section">
-        <h1 class="systems-title">Systems Available</h1>
-        <ng-container *ngTemplateOutlet="systems"></ng-container>
+        <div class="links">
+          <div class="need-help">
+            <a (click)="onShowNeedHelp()">
+              Need Help?
+              <i class="material-icons-outlined">arrow_circle_right</i></a
+            >
+            <div class="show-need-help" [@expandAnimation]="showNeedHelp">
+              <ng-container *ngIf="showNeedHelp"
+                ><div class="need-help-container">
+                  <ul class="need-help-list">
+                    <li>By email at OneHealthID@gov.bc.ca</li>
+                    <li>By phone at 250-857-1969</li>
+                  </ul>
+                </div></ng-container
+              >
+            </div>
+          </div>
+          <div class="about">
+            <a
+              uiAnchor
+              [href]="'https://www2.gov.bc.ca/gov/content/health/practitioner-professional-resources/onehealthid-service'"
```

Comment:
```suggestion
              href="https://www2.gov.bc.ca/gov/content/health/practitioner-professional-resources/onehealthid-service"
```
No need to add `[]` annotation if we use a plain text constant in HTML